app/src/main/java/com/jp/wasabeef/glide/transformations/MaskTransformation.java

```diff
@@ -16,20 +16,20 @@
  * limitations under the License.
  */
 
+import com.bumptech.glide.Glide;
```

Comment:
This is a third party library, prefer not to make changes here, as it makes it difficult to merge in future changes to this library.